packages/gatsby-source-wordpress/src/steps/create-schema-customization/transform-fields/index.js

```diff
@@ -57,8 +57,8 @@ const excludeField = ({
 }) =>
   // this field wasn't previously fetched, so we shouldn't
   // add it to our schema
-  !fieldOfTypeWasFetched(field.type) ||
-  // this field was excluded on it's parent fields Type
+  (!fieldOfTypeWasFetched(field.type) && fieldName !== `id`) ||
```

Comment:
In some cases the id field was being excluded on the new interface types in wpgql which threw schema errors. This ensures it's never excluded